source/common/common/hash.h

```diff
@@ -57,10 +57,18 @@ class MurmurHash {
    */
   static uint64_t murmurHash2_64(absl::string_view key, uint64_t seed = STD_HASH_SEED);
 
+#ifdef _MSC_VER
+#pragma intrinsic(memcpy)
+#endif
+
 private:
   static inline uint64_t unaligned_load(const char* p) {
     uint64_t result;
+#ifdef _MSC_VER
+    memcpy(&result, p, sizeof(result));
+#else
     __builtin_memcpy(&result, p, sizeof(result));
```

Comment:
Is there any reason for using `__builtin_memcpy` on other platforms? Compiler should do what's best without explicit call to the builtin function.

Reply:
+1 please just use memcpy everywhere.

Reply:
According to the following links, `__builtin_memcpy` was created for optimization purposes:
https://gcc.gnu.org/onlinedocs/gcc/Other-Builtins.html
https://stackoverflow.com/questions/11747891/when-builtin-memcpy-is-replaced-with-libcs-memcpy
Unfortunately, this function cannot be used on Windows.

Following your comments, I will replace `__builtin_memcpy` with `memcpy`.